.github/project.yml

```diff
@@ -2,4 +2,6 @@ name: Java Operator SDK Extension
 release:
   current-version: 7.0.0.CR1
   next-version: 7.0.0-SNAPSHOT
-
+  quarkus-platform-branches:
+    - "main"
+    - "3.17"
```

Comment:
@metacosm these don't need to be strings, but otherwise we will get warnings that we mix strings and numbers. I mean, it can also be without quotes like this if preferred:

```yaml
  quarkus-platform-branches:
    - main
    - 3.17
```

Reply:
Probably better with quotes.